google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/stub/EnhancedBigtableStub.java

```diff
@@ -132,6 +132,10 @@ public static EnhancedBigtableStub create(EnhancedBigtableStubSettings settings)
     ServerStreamingCallable<ReadRowsRequest, RowT> withContext =
         filtering.withDefaultCallContext(clientContext.getDefaultCallContext());
 
+    // NOTE: Ideally `withDefaultCallContext` should be the outer-most callable, however the
+    // ReadRowsUserCallable overrides the first() method which be lost. At some point in the future,
```

Comment:
Sorry, grammar to fix (do you want "which would be lost"?)

Reply:
No, I'm sorry. My writing is terrible, thanks for catching that.